server/settings.go

```diff
@@ -69,5 +70,5 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 		notifications = settingOn
 	}
 
-	return p.responsef(header, "Settings updated.\n\t%s notifications %s.", strings.Title(args[1]), notifications)
+	return p.responsef(header, "Settings updated.\n\t%s notifications %s.", cases.Title(language.Und, cases.NoLower).String(args[1]), notifications)
```

Comment:
need for cases and language import here? find something where we don't require 2 different packages for this statement

Reply:
This is only how we can do.

Reply:
What is the purpose of doing this?

Reply:
In `Go 1.18` `strings.Title(str)` was deprecated, should change to `cases.Title(language.Und, cases.NoLower).String(str)`. You can look at here in the documenttion [strings.Title](https://pkg.go.dev/strings#Title)](https://pkg.go.dev/strings#Title)